dev-libs/soci/soci-4.0.1.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Simple Open (Database) Call Interface - write SQL in C++"
+HOMEPAGE="https://github.com/SOCI/soci"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+IUSE="boost firebird mysql odbc oracle postgres +sqlite test"
+RESTRICT="!test? ( test )"
```

Comment:
Try to stick to the ebuild skeleton as much as possible, i.e. IUSE and RESTRICT below KEYWORDS: https://wiki.gentoo.org/wiki/Basic_guide_to_write_Gentoo_Ebuilds

Reply:
fixed